test/catch2/live_restore/api/test_live_restore_fs_directory_list.cpp

```diff
@@ -55,6 +55,17 @@ directory_list(live_restore_test_env &env, const std::string &directory = NO_DIR
     return found_files;
 }
 
+/* Return true if the set contains the full subset. */
+static bool
+contained_by(lr_files set, lr_files subset)
```

Comment:
`subset` is missing cases that `==` covered, for example if `directory_list` incorrectly returns stop files.
What if we manually stripped out the turtle and metadata files?
```cpp
static void require_lr_files_equals(const lr_files& set, const lr_files& subset) {
    lr_files local_set = set;
    lr_files local_subset = subset;

    // Ignore metadata files created by conn_open. We only want to check files that were created by our test scenario.
    local_set.erase("WiredTiger.wt");
    local_set.erase("WiredTiger.turtle");
    local_subset.erase("WiredTiger.wt");
    local_subset.erase("WiredTiger.turtle");

    REQUIRE(local_set == local_subset);
}
```

Reply:
b279c8929